app/src/main/java/com/worldsnas/starplayer/view/musics_list/MusicsListAdapter.kt

```diff
@@ -30,7 +32,25 @@ class MusicListItemViewHolder(private val mBinding: ItemMusicBinding) :
     override fun onBind(obj: Music) {
         mBinding.txtMusicTitle.text = obj.title
         mBinding.txtArtist.text = obj.artist
+
+        sendToPlay(obj)
+    }
+
+    private fun sendToPlay(obj: Music) {
+        mBinding.root.setOnClickListener {
+            val bundle = bundleOf(
```

Comment:
lotfan in karo dakhele fragment anjma bedid
general tip: lotfan kare ezafi toye viewholder hato anjam nadid 
dir ya zod pashimon mishid

Reply:
chtor bayad ba ye function bargardonm on music mored nazar ro?

Reply:
mitonid ye interface to adapter bzarid va to fragment azesh estefadeh konid